modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/BaseOrderItemResourceImpl.java

```diff
@@ -752,9 +753,33 @@ public void create(
 			"createStrategy", "INSERT");
 
 		if ("UPSERT".equalsIgnoreCase(createStrategy)) {
-			orderItemUnsafeConsumer =
-				orderItem -> putOrderItemByExternalReferenceCode(
-					orderItem.getExternalReferenceCode(), orderItem);
+			String updateStrategy = (String)parameters.getOrDefault(
+				"updateStrategy", "UPDATE");
+
+			if ("UPDATE".equals(updateStrategy)) {
```

Comment:
Is it possible to have an UPSERT value for `createStrategy` and an empty value (or null) for the `updateStrategy`?